locale/en_US/locale.xml

```diff
@@ -349,7 +349,7 @@ Note that the OAI interface can be disabled for this site by editing the OJS sys
 </ul>]]></message>
 
 	<!-- System Information -->
-	<message key="admin.overwriteConfigFileInstructions"><![CDATA[<h4>NOTE!</div>
+	<message key="admin.overwriteConfigFileInstructions"><![CDATA[<h4>NOTE!</h4>
```

Comment:
I thought I had that one fixed some time ago… don't know why it's still in here, but good catch!